src/net.cpp

```diff
@@ -1665,7 +1665,7 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
                 switch (pnode->m_conn_type) {
                     case ConnectionType::INBOUND:
                     case ConnectionType::MANUAL:
-                        break;
+                        continue;
```

Comment:
Unless I'm confused, `break` here only acts within the scope of the switch statement; it doesn't break out of the for-loop.